i18n/en.json

```diff
@@ -4054,6 +4084,7 @@
   "unarchive_channel.confirm": "Confirm UNARCHIVE Channel",
   "unarchive_channel.del": "Unarchive",
   "unarchive_channel.viewArchived.question": "Are you sure you wish to unarchive the **{display_name}** channel?",
+  "undefined": "",
```

Comment:
You can remove all these new ones that have no value

Reply:
These appear to be added by the `make i18n-extract` command. Not too sure why that's happening, but I'm going to remove them and push again.

Reply:
@maxerenberg See if you have `storybook-static` anywhere. Delete it. It's probably gitignored. I got my PR merged for the thing that does `i18n-extract` but we haven't updated master yet (I'm doing that now as I type).

Here is the PR https://github.com/mattermost/mattermost-utilities/pull/49/files . Just delete `storybook-static` and you should be gucci. I had the same problem for a long time before figuring out whats wrong.

Reply:
Master just got updated with the latest hash for mm utilities (https://github.com/mattermost/mattermost-webapp/pull/7639). Pull latest master and see if this still happens

Reply:
Thanks, that worked!